[Firehose] Add warning for upcoming default data stream change (#3359)
elastic/observability-docs: docs/en/aws-firehose/aws-firehose.asciidoc

++++

{ecloud} users can ingest logs directly from AWS Kinesis Data Firehose.

> These logs are ready to use out of the box with all current Elastic AWS integrations.

Reviewer: I might push to a seperate paragraph and rephrase to something even more explicit like 

> All Elastic [integrations](https://www.elastic.co/integrations/data-integrations) are supported, without the need for any agents to be deployed in your AWS account. Logs from AWS Kinesis Data Firehose can be automatically routed to the relevant Elastic integration for popular AWS services with zero additional configuration.

Author: I think we should use `All Elastic AWS integrations` instead?

> All Elastic https://www.elastic.co/integrations/data-integrations?solution=all-solutions&category=aws[AWS integrations] are supported, without the need for any agents to be deployed in your AWS account. Logs from AWS Kinesis Data Firehose can be automatically routed to the relevant Elastic integration for popular AWS services with zero additional configuration.

Reviewer 2: ```suggestion
All Elastic https://www.elastic.co/integrations/data-integrations?solution=all-solutions&category=aws[AWS integrations] are supported without deploying agents to your AWS account. Logs from AWS Kinesis Data Firehose can be automatically routed to the relevant Elastic integration for popular AWS services with no additional configuration.
```
